some stm32f4xx devices have no div-q output on their plli2s

FRASTM opened the pull request:
Some stm32f4 have a PLL I2S of type "st,stm32f411-plli2s-clock" or "st,stm32f4-plli2s-clock" 

The "st,stm32f411-plli2s-clock"  has a div-Q output, the "st,stm32f4-plli2s-clock"  has not
The stm32_clock_control_get_subsys_rate() in the clock_stm32_ll_common.c must consider this difference

This PR is fixing  the build error on stm32f405/f407 or f415/f417 mcu:
```
clock_stm32_ll_common.c:440:47: error: 'STM32_PLLI2S_Q_DIVISOR' undeclared (first use in this function); did you mean 'STM32_PLLI2S_M_DIVISOR'?
  440 |                                               STM32_PLLI2S_Q_DIVISOR);
      |                                               ^~~~~~~~~~~~~~~~~~~~~~

```

FRASTM committed:
drivers: clock control: some stm32f4 have no div-q on there plli2s

Check that the plli2s has a DIV-Q output or not. That depends
on the PLLi2S of some stm32F4xx devices

Signed-off-by: Francois Ramu <francois.ramu@st.com>

FRASTM committed:
include: clock control fix pll i2s for stm32f411 serie

The compatible st_stm32f412_plli2s_clock has been replaced
by the st_stm32f411_plli2s_clock

Signed-off-by: Francois Ramu <francois.ramu@st.com>

titouanc commented:
Tested on my stm32f4-disco (based on the stm32f407), and it works as expected.

Thank you very much @FRASTM !

FRASTM committed:
drivers: clock control of some stm32f4x has no 48MHz from PLL i2s

Remove the LL_RCC_PLLI2S_ConfigDomain_48M for the stm32f4
w/o Q divider on the PLLI2S to configure the PLL48CK

Signed-off-by: Francois Ramu <francois.ramu@st.com>

FRASTM commented:
add a commit to fix build error
```
 /zephyr/zephyr/drivers/clock_control/clock_stm32f2_f4_f7.c:162:9: error: implicit declaration of function 'LL_RCC_PLLI2S_ConfigDomain_48M'; did you mean 'LL_RCC_PLLSAI_ConfigDomain_48M'? [-Werror=implicit-function-declaration]
  162 |         LL_RCC_PLLI2S_ConfigDomain_48M(get_pll_source(),
```
